src/api/gnosis.js

```diff
@@ -139,7 +138,7 @@ export const createEvent = async (event) => {
 
   const eventData = {
     ...event,
-    collateralToken: hexWithPrefix(gnosis.etherToken.address),
+    collateralToken: hexWithPrefix(gnosis.olympiaToken.address),
```

Comment:
This is related to market creation and since we are going to remove it, I decided not to fix that 🙂